twitcher/models/service.py

```diff
@@ -11,13 +10,30 @@
 class Service(Base):
     __tablename__ = 'services'
     id = Column(Integer, primary_key=True)
-    url = Column(Text)
-    name = Column(Text)
-    type = Column(Text)
-    purl = Column(Text)
-    public = Column(Integer)  # sqlite does not support Boolean
-    verify = Column(Integer)  # sqlite does not support Boolean
-    auth = Column(Text)
+    url = Column(String(255), nullable=False)
+    name = Column(String(40), unique=True, index=True, nullable=False)
+    type = Column(String(40))
+    purl = Column(String(255))
+    _verify = Column(Integer)  # sqlite does not support Boolean
+    auth = Column(String(40))
 
+    @property
+    def verify(self):
+        if self._verify == 1:
+            return True
+        return False
 
-Index('name_index', Service.name, unique=True, mysql_length=255)
+    @property
+    def public(self):
+        """Return true if public access."""
+        return self.auth not in ['token', 'cert']
+
+    def to_json(self):
```

Comment:
Maybe call it just `json`.
Most packages call it like so, makes it easy to recursively go through objects and always call `.json()` to generate such  response content-type.

Reply:
done.